lib/build.ts

```diff
@@ -153,65 +151,42 @@ export function build(options: BuildOptions): void {
                   }
                 : undefined;
 
-            // Run `cargo check` on an initial time, if needed
-            if (Settings.RUN_CARGO_CHECK && !_ranCargoCheck) {
-                _ranCargoCheck = true;
-                checkCode(options, buildArgs, buildEnv);
-            }
-
             if (releaseDirExists) {
                 console.log(`🍺  Building Rust code...`);
             } else {
                 // The `release` directory doesn't exist for the specified
-                // target. This is most likely an initial run, so `cross` will
-                // take much longer than usual to cross-compile the code.
+                // target. This is most likely an initial run, so `cargo-zigbuild`
+                // will take much longer than usual to cross-compile the code.
                 //
                 // Print out an informative message that the `build` step is
                 // expected to take longer than usual.
                 console.log(
-                    `🍺  Building Rust code with \`cross\`. This may take a few minutes...`
+                    `🍺  Building Rust code with \`cargo-zigbuild\`. This may take a few minutes...`
                 );
             }
 
             const args: string[] = [
-                'build',
+                'zigbuild',
```

Comment:
FYI: cargo-zigbuild has `build` as an alias of `zigbuild` so `build` will also work when you invoke it using `cargo-zigbuild` instead of `cargo zigbuild`. This should allows you to switch between cross and cargo-zigbuild using a command line flag or environment variable. That is you can support both cross and cargo-zigbuild, see also https://github.com/PyO3/setuptools-rust/pull/213.